Sources/RxCodable/ObservableType+RxCodable.swift

```diff
@@ -17,3 +17,31 @@ public extension ObservableType where E == String {
       .map(type, using: decoder)
   }
 }
+
+public extension ObservableType where E == [String: Any] {
+  public func map<T>(_ type: T.Type, using decoder: JSONDecoder? = nil) -> Observable<T> where T: Decodable {
+    return self
+      .map { dict in try JSONSerialization.data(withJSONObject: dict) }
+      .map(type, using: decoder)
+  }
+}
+
+public extension ObservableType where E: Encodable {
+  public func toDictionary() -> Observable<[String:Any]> {
```

Comment:
1. How about renaming it to `mapDictionary()`?
2. I think we need a parameter of `JSONEncoder`. `map()` takes an instance of `JSONDecoder` as a parameter.
3. Could you please append a whitespace after a colon?
    ```diff
    - [String:Any]
    + [String: Any]
    ```
4. How do you think of supporting mapping json array?

Reply:
1. like toBlocking(), toArray(). toXXX() feels extract some data. toDictionary() is more clear for me, aren't you? ( I understand your side, changing data should start with map... )
2. Yes. It seems necessary
3. my mistake.
4. why not

Reply:
1. It's not about extracting data. It's obviously *mapping* data. I think it would be better to rename it to `mapDictionary()`.
4. Could you write that one?

Reply:
1. then ```toJSONString()``` needs rename to ```mapJSONString()```, too?